The prosemirror-dev-mode support broke my build, help me debug this

### Affected Packages

core

### Version(s)

3.0.0-beta.24

### Bug Description

The change in https://github.com/ueberdosis/tiptap/pull/6527 broke the build of my editor:

```
~/projects/django-prose-editor$ yarn prod
yarn run v1.22.22
$ NODE_ENV=production rslib build
  Rslib v0.10.4

info    build started...
error   Build error: 
File: javascript/esm|.../node_modules/@tiptap/core/dist/chunk-TH7LERQR.js:1:1
  × Module not found: Can't resolve 'crypto' in '.../node_modules/@tiptap/core/dist'
   ╭─[2:19]
 1 │ // ../../node_modules/.pnpm/nanoid@3.3.8/node_modules/nanoid/index.js
 2 │ import crypto from "crypto";
   ·                    ────────
 3 │ 
 4 │ // ../../node_modules/.pnpm/nanoid@3.3.8/node_modules/nanoid/url-alphabet/index.js
   ╰────

 @ ./node_modules/@tiptap/core/dist/esm-7BF7U2WM.js
 @ ./node_modules/@tiptap/core/dist/index.js
 @ ./src/editor.js

Error: "crypto" is a built-in Node.js module and cannot be imported in client-side code.

Solution: Check if you need to import Node.js module.
  - If not needed, remove the import.
  - If needed, use "@rsbuild/plugin-node-polyfill" to polyfill it. (See https://npmjs.com/package/@rsbuild/plugin-node-polyfill)

error   Failed to build.
error   Rspack build failed.
```

Even though I'm not setting `enableDevTools` (and the default is false) the bundler still sees the `import` and therefore wants to descend into `prosemirror-dev-tools`, and of course chokes when encountering `crypto`.

The code is here https://github.com/matthiask/django-prose-editor/ and you don't need Django or anything to run the build, just `yarn` (still using yarn); I'm happy to debug this myself but I don't really know where to start. I have tried excluding the library from the build, but since rspack processes the transpiled JavaScript I couldn't just exclude `prosemirror-dev-mode`, I had to do this:

```javascript
    rspack: {
      resolve: {
        alias: {
          "prosemirror-dev-mode": false,
          crypto: false,
          // See node_modules/@tiptap/core/dist/index.js
          "./esm-7BF7U2WM.js": false,
        },
      },
    },
```

The branch is [here](https://github.com/matthiask/django-prose-editor/compare/mk/tiptap-3.0.0-beta.24).

Thanks in advance!

### Browser Used

Firefox

### Code Example URL

_No response_

### Expected Behavior

I expected the build to succeed and not to produce 800kb of additional JavaScript.

### Additional Context (Optional)

_No response_

### Dependency Updates

- [x] Yes, I've updated all my dependencies.

--- a/.changeset/wise-berries-hide.md
+++ b/.changeset/wise-berries-hide.md
@@ -1,0 +1,5 @@
+---
+'@tiptap/core': patch
+---
+
+migration from prosemirror-dev-tools to prosemirror-dev-toolkit

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -53,8 +53,7 @@
   ],
   "devDependencies": {
     "@tiptap/pm": "workspace:*",
-    "@types/prosemirror-dev-tools": "^3.0.6",
-    "prosemirror-dev-tools": "^4.2.0"
+    "prosemirror-dev-toolkit": "^1.1.8"
   },
   "peerDependencies": {
     "@tiptap/pm": "workspace:*"

--- a/packages/core/src/Editor.ts
+++ b/packages/core/src/Editor.ts
@@ -194,14 +194,14 @@ export class Editor extends EventEmitter<EditorEvents> {
   /**
    * Applies ProseMirror dev tools to the editor instance if enabled and running in a browser environment.
    *
-   * This method dynamically imports the `prosemirror-dev-tools` package and applies it to the current
+   * This method dynamically imports the `prosemirror-dev-toolkit` package and applies it to the current
    * editor view. If the dev tools are not installed, a warning is logged to the console.
    *
    * @private
    * @remarks
    * - Dev tools are only applied if `this.options.enableDevTools` is `true` and the code is running in a browser.
    * - If the editor view is not available, the dev tools are not applied.
-   * - If the `prosemirror-dev-tools` package is missing, a warning is shown in the console.
+   * - If the `prosemirror-dev-toolkit` package is missing, a warning is shown in the console.
    *
    * @returns {void}
    */
@@ -210,17 +210,17 @@ export class Editor extends EventEmitter<EditorEvents> {
       return
     }
 
-    import('prosemirror-dev-tools')
-      .then(({ default: apply }) => {
+    import('prosemirror-dev-toolkit')
+      .then(({ applyDevTools }) => {
         if (!this.editorView) {
           return
         }
 
-        apply(this.editorView)
+        applyDevTools(this.editorView)
       })
       .catch(() => {
-        console.warn('[Tiptap warning]: Devtools are enabled but `prosemirror-dev-tools` is not installed.')
-        console.warn("Install 'prosemirror-dev-tools' as a dev dependency to use the dev tools.")
+        console.warn('[Tiptap warning]: Devtools are enabled but `prosemirror-dev-toolkit` is not installed.')
+        console.warn("Install 'prosemirror-dev-toolkit' as a dev dependency to use the dev tools.")
       })
   }
 

--- a/packages/core/src/types.ts
+++ b/packages/core/src/types.ts
@@ -366,7 +366,7 @@ export interface EditorOptions {
   /**
    * Enable a lazy-loaded Prosemirror DevTools integration.
    *
-   * Requires having the `prosemirror-dev-tools` npm package installed.
+   * Requires having the `prosemirror-dev-toolkit` npm package installed.
    * @type boolean
    * @default false
    * @example

